images/ipfailover/keepalived/lib/config-generators.sh

```diff
@@ -190,13 +190,19 @@ function generate_vrrpd_instance_config() {
 
   [ "${instancetype}" = "master" ] && initialstate="state MASTER"
 
+  if [[ ${preempt} == "nopreempt" ]] ; then
+	  initialstate="state BACKUP"
+	  priority=1
+  fi
+
   local instance_name ; instance_name=$(generate_vrrp_instance_name "${servicename}" "${iid}")
 
   local auth_section ; auth_section=$(generate_authentication_info "${servicename}")
   local vip_section ; vip_section=$(generate_vip_section "${vips}" "${interface}")
   # Emit instance
   echo "
 vrrp_instance ${instance_name} {
+   smtp_alert
```

Comment:
how about providing a parameter to enable/disable notifications, since not everyone wants to receive notifications?

Reply:
Changed it to set when SMTP_SERVER is provided.